internal/plugins/golang/v2/init.go

```diff
@@ -58,51 +58,3 @@ func (p *initPlugin) Run() error {
 
 	return nil
 }
-
```

Comment:
this code was centralized in internal/plugins/sdk since is required for the other plugins as well.